applications/serial_lte_modem/doc/TCPIP_AT_commands.rst

```diff
@@ -0,0 +1,1541 @@
+  * ``1`` - plain text (default value)
+  * ``2`` - JSON
+  * ``3`` - HTML
+  * ``4`` - OMA TLV
+
+* The ``<size>`` value is an integer and it represents the actual number of bytes received.
+  The maximum size for ``NET_IPV4_MTU`` is 576 bytes.
+  It must not have any ``NULL`` character in the middle.
+
+Examples
+~~~~~~~~
+
+::
+
+   AT#XRECV
+   Test OK
+   #XRECV: 1, 7
+   OK
+
+Read command
+------------
+
+The read command is not supported.
+
+Test command
+------------
+
+The test command is not supported.
+
+UDP Send data #XSENDTO
+======================
+
+The ``#XSENDTO`` command allows you to send data over the UDP channel.
+
+Set command
+-----------
+
+The set command allows you to send data over the UDP channel.
+
+Syntax
+~~~~~~
+
+::
+
+   #XSENDTO=<url>,<port>,<datatype>,<data>
+
+* The ``<url>`` parameter is a string and it indicates the hostname or the IP address to connect to.
+  Its maximum size can be 128 bytes.
+  When the parameter is an IP address, it supports IPv4 only, not IPv6.
+* The ``<port>`` parameter is an integer and it represents the port of the TCP service.
+* The ``<datatype>`` parameter can accept one the following values:
+
+  * ``0`` - hexidecimal string (e.g. "DEADBEEF" for 0xDEADBEEF)
+  * ``1`` - plain text (default value)
+  * ``2`` - JSON
+  * ``3`` - HTML
+  * ``4`` - OMA TLV
+
+* The ``<data>`` parameter is a string and it contains the data being sent.
+  The maximum size for ``NET_IPV4_MTU`` is 576 bytes.
+  It must not have any``NULL`` character in the middle.
+
+Response syntax
+~~~~~~~~~~~~~~~
+
+::
+
+   #XSENDTO: <size>
+
+* The ``<size>`` value is an integer and it represents the actual number of bytes sent.
+
+Examples
+~~~~~~~~
+
+::
+
+   AT#XSENDTO="test.server.com",1234,"Test UDP"
+   #XSENDTO: 8
+   OK
+
+Read command
+------------
+
+The read command is not supported.
+
+Test command
+------------
+
+The test command is not supported.
+
+UDP receive data #XRECVFROM
+===========================
+
+The ``#XRECVFROM`` command allows you to receive data through the UDP channel.
+
+Set command
+-----------
+
+The set command allows you to receive data through the UDP channel.
+
+Syntax
+~~~~~~
+
+::
+
+   #XRECVFROM[=<size>]
+
+The ``<size>`` value is an integer and it represents the actual number of bytes requested.
+It is set to match the ``NET_IPV4_MTU`` when not specified.
+
+Response syntax
+~~~~~~~~~~~~~~~
+
+::
+
+   <data>
+   #XRECVFROM: <datatype>, <size>
+
+
+* The ``<data>`` value is a string and it contains the data being received.
+* The ``<datatype>`` parameter can accept one the following values:
+
+  * ``0`` - hexidecimal string (e.g. "DEADBEEF" for 0xDEADBEEF)
+  * ``1`` - plain text (default value)
+  * ``2`` - JSON
+  * ``3`` - HTML
+  * ``4`` - OMA TLV
+
+* The ``<size>`` value is an integer and it represents the actual number of bytes received.
+
+Examples
+~~~~~~~~
+
+::
+
+   AT#UDPRECVFROM="test.server.com",1234
+   Test OK
```

Comment:
Is this the right example?

Reply:
<host> is pseudo-server, as we cannot expose the Nordic test server that we used

Reply:
@junqingzoe To clarify, I mean the example mentions UDPRECVFROM but this is located in the documentation for XRECVFROM. Is this supposed to be like this or it is the wrong example?

Reply:
Let's use example.com for all server URLs (unless we have some test server that actually can be used by everyone).